test/integration/testing/commonjs.test.js

```diff
@@ -63,3 +64,7 @@ testProp(
     t.deepEqual(resultEsm, resultCjs);
   },
 );
+
+test("throwscape (cjs)", (t) => {
+  t.throws(() => new Throwscape(options), { instanceOf: Error });
+});
```

Comment:
Upgrade this to a [generative test](https://github.com/ericcornelissen/shescape/blob/ced66ae3d908b76f5f56886be7e4d15ce888f853/CONTRIBUTING.md#generative-tests) for arbitrary `options`. (Also, this test is currently incorrect because `options` is undefined.)